### ISSUE TITLE
Date for branch, time for exp rows

### ISSUE THREAD

**Issue by maxagin** [iterative/vscode-dvc#2050]
DVC has date only for the branch, the expS have the time. Why can not we do the same?

<img width="648" alt="Screen Shot 2022-07-15 at 11 26 29 PM" src="https://user-images.githubusercontent.com/98249521/179337646-b01d81d8-003e-4092-a5b4-646e6042ece0.png">

**Comment by shcheklein** [comment]
Max, sorry, closing this since it's not actionable . What do those dates mean? What do we actually want from those dates (what problem do they solve?), etc - the ticket ideally should be described like that. Please feel free to reopen if you are able to provide more context / information to this.

**Comment by maxagin** [comment]
@shcheklein , the screenshot in the description is from DVC (or I am wrong?). 
My question is why does DVC have it one way and VSCode, another way? That is it. Thanks

**Comment by shcheklein** [comment]
@maxagin yep, my point I guess is for this ticket to be more or less actionable - let's ask the DVC, put together some description, some logic behind it, etc. I would make it easier to say if it makes sense or not.

**Comment by mattseddon** [comment]
Should be combined with #1444

**Comment by mattseddon** [comment]
and https://github.com/iterative/vscode-dvc/issues/2031

**Comment by maxagin** [comment]
> @maxagin yep, my point I guess is for this ticket to be more or less actionable - let's ask the DVC, put together some description, some logic behind it, etc. I would make it easier to say if it makes sense or not.

Hello @dberenbaum , would you be so kind to clarify for us the logic behind the information on the screenshot in the description of this ticket? Thank you Dave!

**Comment by maxagin** [comment]
Thank you Dave for the clarification!

**Comment by maxagin** [comment]
> Show times for today's experiments since they are the most recent and important to compare at a granular level

@dberenbaum  how does actually the time will help to compare EXPs (or checkpoints)? 
Maybe users will more likely be interested to know the range of time one run took?

Q: Do the checkpoints of one EXP, can have different dates?

**Comment by dberenbaum** [comment]
> @dberenbaum how does actually the time will help to compare EXPs (or checkpoints)?

It doesn't help compare them, but it sometimes helps organize them. Seeing that this experiment was run a week ago might help me remember something about it or see quickly that it's not part of my current set of experiments. It's less important for different times on the same day, but it still helps me see the order in which I ran the experiments, which can help organize them in my mind at least.

> Maybe users will more likely be interested to know the range of time one run took?

Yes, the time it took to run an experiment could be helpful. We have discussed adding that to DVC (https://github.com/iterative/dvc/discussions/7160) but right now there's no functionality to provide that info.
